notebook/Clustering Flag Data - Pipeline.ipynb

```diff
@@ -64,10 +64,8 @@
   },
```

Comment:
<p>Looks like you're committing an error?</p>

 _Reply via <a href='https://app.reviewnb.com/georgetown-analytics/machine-learning/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I will clear the output, We are using: Clustering Flag Data.ipynb instead of Clustering Flag Data - Pipeline.ipynb